packages/data-context/test/unit/sources/fixtures/graphqlFixtures.ts

```diff
@@ -45,51 +94,82 @@ export const CLOUD_PROJECT_REQUEST_WITH_VARIABLES = parse(`
 
 export const FAKE_SHAS = ['fcb90f', 'fc753a', '207e80']
 
-export const FAKE_PROJECT_NO_RUNS = { data: { cloudProjectBySlug: { __typename: 'CloudProject', runsByCommitShas: [] } } }
+export const FAKE_PROJECT_WITH_ERROR = {
+  error: {
+    name: 'FAKE ERROR',
+    message: 'This had an error',
+  },
+}
 
-export const FAKE_PROJECT_ONE_RUNNING_RUN = { data: { cloudProjectBySlug: { __typename: 'CloudProject', runsByCommitShas: [{ runNumber: 1, status: 'RUNNING', commitInfo: { sha: FAKE_SHAS[0] } }] } } }
+export const FAKE_PROJECT_NO_RUNS = {
+  data: {
+    cloudProjectBySlug: { __typename: 'CloudProject', runsByCommitShas: [] },
+  },
+}
 
-export const FAKE_PROJECT_MULTIPLE_COMPLETED = { data: { cloudProjectBySlug: { __typename: 'CloudProject', runsByCommitShas: [
-  { runNumber: 4, status: 'FAILED', commitInfo: { sha: FAKE_SHAS[1] } }, { runNumber: 1, status: 'PASSED', commitInfo: { sha: FAKE_SHAS[0] } },
-] } } }
+export const FAKE_PROJECT_ONE_RUNNING_RUN = {
+  data: {
+    cloudProjectBySlug: {
+      __typename: 'CloudProject',
+      runsByCommitShas: [
+        { runNumber: 1, status: 'RUNNING', commitInfo: { sha: FAKE_SHAS[0] } },
+      ],
+    },
+  },
+}
 
-export const FAKE_PROJECT_MULTIPLE_COMPLETED_PLUS_RUNNING = { data: { cloudProjectBySlug: { __typename: 'CloudProject', runsByCommitShas: [{ runNumber: 5, status: 'RUNNING', commitInfo: { sha: FAKE_SHAS[2] } }, { runNumber: 4, status: 'FAILED', commitInfo: { sha: FAKE_SHAS[1] } }, { runNumber: 1, status: 'PASSED', commitInfo: { sha: FAKE_SHAS[0] } }] } } }
+export const FAKE_PROJECT_MULTIPLE_COMPLETED = {
+  data: {
+    cloudProjectBySlug: {
+      __typename: 'CloudProject',
+      runsByCommitShas: [
+        { runNumber: 4, status: 'FAILED', commitInfo: { sha: FAKE_SHAS[1] } },
+        { runNumber: 1, status: 'PASSED', commitInfo: { sha: FAKE_SHAS[0] } },
+      ],
+    },
+  },
+}
 
-export const FAKE_PROJECT_ONE_RUNNING_RUN_ONE_SPEC = {
+export const FAKE_PROJECT_MULTIPLE_COMPLETED_PLUS_RUNNING = {
   data: {
     cloudProjectBySlug: {
       __typename: 'CloudProject',
-      current: {
-        runNumber: 1,
-        completedInstanceCount: 1,
-        totalInstanceCount: 1,
-        totalTests: 5,
-        status: 'RUNNING',
-      },
+      runsByCommitShas: [
+        { runNumber: 5, status: 'RUNNING', commitInfo: { sha: FAKE_SHAS[2] } },
+        { runNumber: 4, status: 'FAILED', commitInfo: { sha: FAKE_SHAS[1] } },
+        { runNumber: 1, status: 'PASSED', commitInfo: { sha: FAKE_SHAS[0] } },
+      ],
     },
   },
 }
```

Comment:
Not reltaed to this file but I found out that

<img width="321" alt="image" src="https://user-images.githubusercontent.com/19196536/227883609-ae7b62a2-5f97-450c-abab-1120615ebcca.png">

When I am on the **spec page** this will show the old status until the new run is completed done - eg, if my run takes 10 min, even if the first test fails, I won't know until the entire run finishes. Is that correct, or should this start auto upadating?  Potential bug in polling logic for the Sidebar subscription?

The updates on the debug page are working as expected.

Reply:
> Not reltaed to this file but I found out that
> 
> <img alt="image" width="321" src="https://user-images.githubusercontent.com/19196536/227883609-ae7b62a2-5f97-450c-abab-1120615ebcca.png">
> 
> When I am on the **spec page** this will show the old status until the new run is completed done - eg, if my run takes 10 min, even if the first test fails, I won't know until the entire run finishes. Is that correct, or should this start auto upadating? Potential bug in polling logic for the Sidebar subscription?
> 
> The updates on the debug page are working as expected.

This is the expected behavior at this time.  There is another initiative called [In-App Run Notifications](https://app.clickup.com/t/18033298/PM-3555) to address providing better feedback to the user about Cloud runs when you are not on the Debug page or not looking at the Cypress app at all.